client/src/APIs/hatchways.js

```diff
@@ -0,0 +1,5 @@
+import axios from 'axios';
+
+export default axios.create({
+	baseURL: 'http://localhost:3001/api',
+});
```

Comment:
this shouldn't be necessary with the way the project template is set up.

Reply:
I created the APIs folder considering that we were going to make use of some other APIs. I just removed the file and modified the API call on the interview page.